kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/MongoKVTable.java

```diff
@@ -72,11 +73,11 @@ public MongoKVTable(final MongoClient client, final String name) {
     );
 
     final MongoDatabase database = client.getDatabase(name).withCodecRegistry(pojoCodecRegistry);
-    genericDocs = database.getCollection(name);
-    docs = database.getCollection(name, KVDoc.class);
+    genericDocs = database.getCollection(KV_COLLECTION_NAME);
+    docs = database.getCollection(KV_COLLECTION_NAME, KVDoc.class);
 
-    genericMetadata = database.getCollection(name + METADATA_COLLECTION_SUFFIX);
-    metadata = database.getCollection(name + METADATA_COLLECTION_SUFFIX, MetadataDoc.class);
+    genericMetadata = database.getCollection(METADATA_COLLECTION_NAME);
+    metadata = database.getCollection(METADATA_COLLECTION_NAME, MetadataDoc.class);
```

Comment:
these lines are the "change" right? Just want to make sure it doesn't affect existing C* deployments

Reply:
yes, sorry, to clarify this only affects mongo

Reply:
The only thing that would potentially affect C* as well is the new length check -- but no existing deployments are past that limit. And anyways, I think C* actually does have a length limit as well, though it was more ambiguous